shell/packages/sandstorm-db/db.js

```diff
@@ -430,6 +432,29 @@ ApiTokens = new Mongo.Collection("apiTokens");
 //   ...
 // }
 
+ApiHosts = new Mongo.Collection("apiHosts");
+// Allows defining some limited static behavior for an API host when accessed unauthenticated. This
+// mainly exists to allow backwards-compatibility with client applications that expect to be able
+// to probe an API host without authentication to determine capabilities such as DAV protocols
+// supported, before authenticating to perform real requests. An app can specify these properties
+// when creating an offerTemplate.
+//
+// Each contains:
+//   _id:          apiHostIdHashForToken() of the corresponding API token.
+//   hash2:        hash(hash(token)), aka hash(ApiToken._id). Used to allow ApiHosts to be cleaned
```

Comment:
Oh, I see.  It removes all of the ApiTokens associated with that grain that are either owned by that user or by `{ webkey: null }`.

I thought all ApiTokens were left lying around because if you share a grain, the ApiToken owned by the person you shared with definitely is still around after you delete the grain, but I see that's not the case in general. :)